tests/utils/npm-utils.spec.js

```diff
@@ -188,14 +191,11 @@ describe("npmUtils", () => {
         it("should log an error message if npm throws ENOENT error", async () => {
             const logErrorStub = sinon.spy();
             const npmUtilsStub = sinon.stub(spawn, "sync").returns({ error: { code: "ENOENT" } });
+            const log = await import("../../lib/utils/logging.js");
 
-            const { installSyncSaveDev: stubinstallSyncSaveDev } = await esmock("../../lib/utils/npm-utils.js", {
-                "../../lib/utils/logging.js": {
-                    error: logErrorStub
-                }
-            });
+            sinon.replaceGetter(log, "error", () => logErrorStub);
```

Comment:
This replaces a property on the module namespace object?

Reply:
Yes, do you know of a better way to do that?

Reply:
Nope, I just don't understand how it can work, shouldn't module namespace objects be frozen?

Reply:
I think that Vitest does some kind of bundling or transpiling, which is also why the unit tests cannot be debugged.

Reply:
Yeah, it seems that when running through Vitest this isn't a real namespace object.